packages/babel-core/src/config/files/plugins.js

```diff
@@ -19,6 +19,8 @@ const OTHER_PLUGIN_ORG_RE = /^(@(?!babel\/)[^/]+\/)(?![^/]*babel-plugin(?:-|\/|$
 const OTHER_PRESET_ORG_RE = /^(@(?!babel\/)[^/]+\/)(?![^/]*babel-preset(?:-|\/|$)|[^/]+\/)/;
 const OTHER_ORG_DEFAULT_RE = /^(@(?!babel$)[^/]+)$/;
 
+const nodePaths = process.env("NODE_PATH").split(":");
```

Comment:
On windows it uses `;`

Reply:
Ah, thanks. The latter I figured in the meantime.
As for the former, I am in the woods. I do not use Windows for like 4 years now.
Anyway, is there a *preferred* way to detect platform?
Would `os.type().toLowerCase().indexOf("win") === 0` do? Or is there any global constant I should be aware of?

Reply:
You can use `.split(path.delimiter)` (https://nodejs.org/api/path.html#path_path_delimiter)